certified-connectors/CyberProof/apiDefinition.swagger.json

```diff
@@ -184,7 +184,7 @@
                     "value-path": "ref",
```

Comment:
@sandeepkokare94 you have 'x-ms-dynamic-values' extension references properties within parameters. You need to define the 'x-ms-dynamic-list' extension. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions#how-to-use-dynamic-values)